.github/actions/run-eden-test/action.yml

```diff
@@ -20,6 +22,14 @@ runs:
       with:
         file_system: ${{ inputs.file_system }}
         tpm_enabled: ${{ inputs.tpm_enabled }}
+    - name: Setup eve version
+      run: |
+        if [[ -z ${{ inputs.eve_tag }} ]]; then
```

Comment:
what about using if in GitHub actions, we don't need empty ./eden config set default

Reply:
And let's just propagate eve_tag parameter to setup-environment action